resources/js/Pages/Mentor.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import MentorForm from "@/Organisms/MentorForm";
+import FormLayout from "@/Atoms/FormLayout";
+
+export default function Index() {
+    return (
+        <div>
+            <FormLayout
+                heading="Mentor Form"
+                message="Please fill out the questionnaire below if you are interested in finding someone to mentor you"
```

Comment:
This text is the same as on the Mentee page... Should this say "...finding someone to mentor" or something like that?
```suggestion
                message="Please fill out the questionnaire below if you are interested in finding someone to mentor"
```

Reply:
Yes, I thought I had changed it.. good point.